client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -309,7 +309,7 @@ export default React.createClass( {
 			<SelectDropdown compact
 					className="plugin-list-header__actions_dropdown"
 					key="plugin-list-header__actions_dropdown"
-					selectedText={ this.translate( 'Actions' ) } >
+					selectedText={ this.translate( 'Actions', { context: 'context added' }) } >
```

Comment:
:disappointed: 92 existing translations will be lost with this change.
If the string must change, the following option has already been translated **40** times:
`translate( 'Action:' )` *ES Score: 5.94*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).